src/seismometer/_api.py

```diff
@@ -1058,7 +1243,157 @@ def show_cohort_summaries(by_target: bool = False, by_score: bool = False):
     """
     dfs = _get_cohort_summary_dataframes(by_target, by_score)
 
-    display(template.render_cohort_summary_template(dfs))
+    return template.render_cohort_summary_template(dfs)
+
+
+# endregion
+
+# region Exploration Widgets
+
+
+@export
+class ExploreModelEvaluation(ExplorationModelSubgroupEvaluationWidget):
+    """
+    A widget for exploring the model performance of a cohort.
+    """
+
+    def __init__(self):
+        """
+        Exploration widget for model evaluation.
+
+        This includes the ROC, recall vs predicted condition prevalence, calibration,
+        PPV vs sensitivity, sensitivity/specificity/ppv, and a histogram.
+        """
+        from seismometer._api import plot_model_evaluation
+
+        super().__init__("Model Performance", plot_model_evaluation)
+
+
+@export
+class ExploreCohortEvaluation(ExplorationCohortSubclassEvaluationWidget):
+    """
+    A widget for exploring the model performance of a cohort.
```

Comment:
Can we get this first line of the docstring to be a little more specific; the goal being to help a user know which they want to use from the sphinx TOC:
![image](https://github.com/user-attachments/assets/6ba48d95-eb78-4159-86ba-c85577662bf8)

And note that the table pictured only displays that first-line summary. On following the link you then see the anything beyond the first blankline of the class docstring (and of course the constructor, methods, etc)

Reply:
![image](https://github.com/user-attachments/assets/aa572e6e-c755-4138-8fef-0683f3459bd8)

Thanks for the suggestion!